data/src/main/java/com/antosik/benchproject/data/movies/common/api/MoviesApi.kt

```diff
@@ -1,16 +1,16 @@
 package com.antosik.benchproject.data.movies.common.api
 
-import com.antosik.benchproject.data.common.constants.Constants
 import com.antosik.benchproject.data.movie.details.entity.MovieDetailsRemote
 import com.antosik.benchproject.data.popular.movies.entity.MovieResponseRemote
 import retrofit2.http.GET
 import retrofit2.http.Path
+import retrofit2.http.Query
 
 internal interface MoviesApi {
 
-    @GET("popular?api_key=" + Constants.API_KEY)
-    suspend fun getPopularMovies(): MovieResponseRemote
+    @GET("popular?")
```

Comment:
I think that `?` is redundant, it will be added by retrofit

Reply:
Yes, it's redundant, therefore I deleted it 😄